### PR TITLE
[ci skip] Update Readme

### PR DESCRIPTION
**Pull request by odidukh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,9 +37,59 @@ Find more details in the [docs directory](docs).
 
 Local development setup:
 
-* [Docker compose](https://github.com/rubykube/peatio-workbench)
-* [on Mac OS X](docs/setup-osx.md)
-* [on Ubuntu](docs/setup-ubuntu.md)
+* ### [Docker compose](https://github.com/rubykube/peatio-workbench)
+We suggest you to start using Peatio by installing Workbench. Workbench which is based on [Docker containers](https://www.docker.com/what-docker) is a conveniet and straightforward way to start Peatio development environtment.
```
Comment (mod) [inline README.md]: ## Quick Install

Rubykube Peatio, uses a microservice architecture, and is composed of several services, The easiest way to install and evaluate peatio is to use the workbench.

Our Peatio use docker containers which are built and maintained publicly : https://hub.docker.com/r/rubykube
Workbench use a simple tool called docker-compose.

1. Install Docker on your computer: [Docker](https://docs.docker.com/install/)
2. Install Docker compose [Docker compose](https://docs.docker.com/compose/install/)

### Start Peatio with workbench

1. Recursive clone : `git clone --recursive https://github.com/rubykube/workbench.git`
2. Move to workbench `cd workbench`
3. Build the images: `make build`
4. Run the application: `make run`

This setup use a load balance configured with the following hosts to install in `etc/hosts`
```
0.0.0.0 api.wb.local
0.0.0.0 auth.wb.local

0.0.0.0 api.slanger.wb.local
0.0.0.0 ws.slanger.wb.local

0.0.0.0 pma.wb.local
0.0.0.0 monitor.wb.local
+```

**Comment by ianeinser** [comment]
Hi, 
is it possible to get the latest stable versions of apps in /app/? some of the apps in /app are old versions. Thanks

**Comment by shal** [comment]
@odidukh, `[ci slip]` looks wrong.

**Comment by AboShanab** [comment]
@odidukh There's some typo's on this pull request:
Under Quick Install text:
4.  [Vagrant](https://www.vagrantup.com/downloads.html) -> 3.  [Vagrant](https://www.vagrantup.com/downloads.html)

Under Barong text:
[peatio.teck](https://peatio.tech/#/) -> [peatio.tech](https://peatio.tech/#/)

Under Start peatio with workbench text:
2. Build the images: `make build` -> 3. Build the images: `make build`
3. Run the application: `make run` -> 4. Run the application: `make run` 

'Simply start your local server. Now you're able to log in with your local Barong and Peatio.'
This sentence isn't required as well, we are actually running the Peatio by doing 'Run the application' command, also what's the difference between [Peatio](https://hub.docker.com/r/rubykube/peatio/) docker and workbench? it looks the same..

File: README.md [hunk 2]
```diff
@@ -37,9 +37,52 @@ Find more details in the [docs directory](docs).
...
+
+0.0.0.0 api.slanger.wb.local
+0.0.0.0 ws.slanger.wb.local
+
+0.0.0.0 pma.wb.local
+0.0.0.0 monitor.wb.local
+```
+
+Now you have peatio up and running.
+
```
Comment (mod) [inline README.md]: you can remove this chunk

Comment (mod) [reply README.md]: it doesn't mean anything and workbench starts everything

File: README.md [hunk 3]
```diff
@@ -78,7 +121,23 @@ You must know what you're doing, there's no shortcut. Please get prepared before
 * Highly configurable and extendable
 * Industry standard security out of box
 * Maintained by [peatio.tech](https://www.peatio.tech)
-* [KYC Verification](http://en.wikipedia.org/wiki/Know_your_customer) provided by [Barong](https://www.github.com/rubykube/peatio)
+* [KYC Verification](http://en.wikipedia.org/wiki/Know_your_customer) provided by [Barong](https://www.github.com/rubykube/barong)
+
+## Barong
+
+Barong is an important part of [peatio.teck](https://peatio.tech/#/) stack. Barong provides an OAuth server specially written for our service.
```
Comment (mod) [inline README.md]: link is peatio.tech

Comment (mod) [reply README.md]: can you give a feature list of Barong?

**Review by mod — CHANGES_REQUESTED** [review]

**Comment by mod** [comment]
@odidukh Can you add the installation steps using : 
`docker-compose -f config/backend.yml up -d`

Can you also remove the dead links  ?

**Comment by odidukh** [comment]
In which place I should add this command `docker-compose -f config/backend.yml up -d`?

**Comment by ysv** [comment]
Progress in #1551

**Pull request closed without merging**